### PR TITLE
Done setup project

### PR DESCRIPTION
**Pull request by dinhthi12**
## Related Tickets:
- [#1537Apr10](https://edu-redmine.vn/issues/xxxx)

## WHAT this PR does?

- Done setup project for team.

## Notes Impacted Areas:
- N/A.

## Attachments:
- 
Screenshot from 2024-04-10 15-38-17
.

### CODE REVIEW

**Comment by hoangpt1310** [comment]
OK

File: database/migrations/2014_10_12_000000_create_users_table.php
Comment (hoangpt1310) [inline database/migrations/2014_10_12_000000_create_users_table.php:1]: Pull sao làm database đúng không

Comment (dinhthi12) [reply database/migrations/2014_10_12_000000_create_users_table.php:1]: đúng rồi fe

Comment (tn0810520) [reply database/migrations/2014_10_12_000000_create_users_table.php:1]: Ok

**Pull request merged**